app/models/education_benefits_claim.rb

```diff
@@ -121,7 +121,27 @@ def form_matches_schema
     return unless form_is_string
     return unless FORM_TYPES.include?(form_type)
 
-    errors[:form].concat(JSON::Validator.fully_validate(FORM_SCHEMAS[form_type], parsed_form))
+    errors[:form].concat(JSON::Validator.fully_validate(get_form_schema, parsed_form))
+  end
+
+  def get_form_schema
+    schema = FORM_SCHEMAS[form_type]
+
+    if is_1995?
+      schema = schema.deep_dup
+      required = []
+
+      required <<
```

Comment:
Yeah, the library seems to force us to write a custom field when anyOf is used, but I didn't know you could use anyOf for required like that. I think that would actually break how the library displays required fields.

That said, we're probably going to be pulling in the schema in piecemeal on the front end, so it's possible to skip things like this if we need to.